docs/administration/configuring-replication/create-replication-rules.md

```diff
@@ -57,7 +61,7 @@ A replication endpoint must exist before you create a replication rule. To creat
    You can filter images for replication based on the labels that are applied to the images. However, changing a label on an image does not trigger replication. Event-based replication is limited to pushing, retagging, and deleting images.
    {{< /note >}}
 
-   ![Trigger mode](../../../img/replication-rule5.png)
+   ![Trigger mode](../../img/replication-rule5.png)
```

Comment:
../../../ should be correct, it should be one more level than the actual folder structure, the image has its own directory basically, please double check